lib/index.iife.js

```diff
@@ -1,3 +1,4 @@
+var global = typeof globalThis !== 'undefined' ? globalThis : global || self; // from umd
```

Comment:
```suggestion
var _global = typeof globalThis !== 'undefined' ? globalThis : global || self; // from umd
```

We need to use a different name to avoid circular reference

Reply:
copy that! I commited.